config/initializers/secret_token.rb

```diff
@@ -5,3 +7,4 @@
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
 Calculators::Application.config.secret_token = '9d7d2b0f5e683258253e0d96d7944de860e9a0ccd61095943260906e6f6fa6d4d7a44d7d0f6157ae4c5494e5546ef4deb41c5fdf9c75f5abc1ef6c1f9981e9a3'
+Calculators::Application.config.secret_key_base = '4ccd058460cfede44927db9b91a4c67e03742bbe7c5e120709e60acb94e180f6499cb8ca7819905f7c0fff008c9da675d5879d20fbdba91dcadb9ae437761c56786'
```

Comment:
Following in the footsteps of all the other Rails 4 upgrades, here's the required alphagov-deployment PR that sets this token to something else upon deploy. :sunflower: https://github.gds/gds/alphagov-deployment/pull/894